Stripe/STPAddress.m

```diff
@@ -83,12 +83,12 @@ - (BOOL)containsRequiredFields:(STPBillingAddressFields)requiredFields {
 }
 
 - (BOOL)hasValidPostalAddress {
-    return self.line1.length > 0 &&
-    self.city.length > 0 &&
-    self.state.length > 0 &&
-    self.country.length > 0 &&
-    [STPPostalCodeValidator stringIsValidPostalCode:self.postalCode 
-                                        countryCode:self.country];
+    return (self.line1.length > 0 
```

Comment:
I added a bunch of new checks to STPAddressTests. `containsRequiredFields:` with `Full` as the type calls through to hasValidPostalAddress so this should be covered well now.